### ISSUE TITLE
[1.13.2 - Beta-2] Dynmmap only loads when world updates occur

### ISSUE THREAD

**Issue by thinknoodles000** [webbukkit/dynmap#2390]
Dynmmap only loads when world updates occur, Except 2 chunks that had a giant piston in it, other than that it DOESN'T load

**Comment by otlet** [comment]
I have the same problem. I wanted to add a map, fullrender to the existing built-up map, and unfortunately it only rendered me the place where someone had been installed since DynMap was installed.

**Comment by MalowZer** [comment]
Indeed it's seems like it's a common error for the 1.13.2 at least. I've tried many many differents things, even modify many options in the configuration files but nothing changes. Hope they will fix this soon, don't want to walk on every chunk of the map, it's a bit time waster ! 

if btw someone got a plugin that can reproduce the behavior of a player walking across the map, it would be a good alternative for the moment !

**Comment by thinknoodles000** [comment]
> Do you mean that your map is rendering only chunk that are traveled by players ? even if they are already existing right ?

no, I literly mean when chunk updates occur, Like placing a block, But im not 100% sure, Dont take my word on it http://prntscr.com/lnaqyk All these chunks have been updated at there own times, Like a block being repalced, When the server restarts, It continues it load ONLY those chunks, Unless i update more

**Comment by thinknoodles000** [comment]
> > Do you mean that your map is rendering only chunk that are traveled by players ? even if they are already existing right ?
> 
> no, I literly mean when chunk updates occur, Like placing a block, But im not 100% sure, Dont take my word on it http://prntscr.com/lnaqyk All these chunks have been updated at there own times, Like a block being repalced, When the server restarts, It continues it load ONLY those chunks, Unless i update more

I tried to go to other chunks, doesn't seen to load anything, like i said, its JUST when the chunk updates

**Comment by thinknoodles000** [comment]
http://prntscr.com/lnatr8
Also wtf is this? this picture shows 2 same chunks duplicated

**Comment by mikeprimm** [comment]
Problem is a spigot issue, from what I can see

> http://prntscr.com/lnatr8
> Also wtf is this? this picture shows 2 same chunks duplicated

You have two different resolution there - one was rendered with a lower res, one with a higher.  Changing resolutions also changes offsets of tiles, relative to the world origin (xyz = 0,0,0).

**Comment by mikeprimm** [comment]
OK - I've coded what I think is a valid workaround for the Spigot issue (the chunk loading call I use, loadChunk(x, z, false), to load chunks without triggering generation of the chunks that don't exist yet, is failing for chunks that were generated pre-1.13 AND have not yet been loaded and migrated to the 1.13 format).  The workaround isn't perfect, but I've confirmed that it seems to work well with a 1.13.2 server using a 1.12.2-generated world.   See http://dynmap.us/builds/dynmap/Dynmap-3.0-SNAPSHOT-spigot.jar for the dev build.
